### PR TITLE
Spelling & Grammar

### PR DESCRIPTION
**Pull request by michaeldclifford**
Minor spelling and grammatical edits.

### CODE REVIEW

File: docs/safe_lambda_deployments.rst [hunk 1]
```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
 ---------------------------------------------
 
 Every Lambda function can have any number of Versions and Aliases
-associated with them. Versions are immutable snapshot of function
+associated with them. Versions are immutable snapshots of a function
 including code & configuration. If you are familiar with git, they are
```
Comment (ombozek) [inline docs/safe_lambda_deployments.rst]: s/they/versions

File: docs/safe_lambda_deployments.rst [hunk 2]
```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
...
-similar to commits. It is a good practice in general to publish a new
+similar to commits. In general, it is a good practice to publish a new
 version every time you update your function code. When you invoke a
-specific version (using function name + version number combination) you
-are guaranteed to get the same code & configuration irrespective of
+specific version (using the function name + version number combination) you
+are guaranteed to get the same code & configuration irrespective of the
```
Comment (ombozek) [inline docs/safe_lambda_deployments.rst]: s/irrespective/regardless

File: docs/safe_lambda_deployments.rst [hunk 3]
```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
...
 state of the function. This protects you against accidentally updating
 production code.
 
 To effectively use the versions, you should create an Alias which is
```
Comment (ombozek) [inline docs/safe_lambda_deployments.rst]: To effectively use versions you can create an alias which will act as a pointer to a specific version

Comment (ombozek) [reply docs/safe_lambda_deployments.rst]: They aren't 'literally' a pointer... It's frustrating that the second definition of literally is figuratively 
https://www.merriam-webster.com/dictionary/literally

File: docs/safe_lambda_deployments.rst [hunk 4]
```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
...
-to the function and accepted by the Invoke APIs. If you invoke an Alias,
+to the function and are accepted by the Invoke APIs. If you invoke an Alias,
 Lambda will in turn invoke the version that the Alias is pointing to.
 
 In production, you will first update your function code, publish a new
-version, invoke the version directly to run tests against it, and after
-you are satisfied flip the Alias to point to the new version. Traffic
-will instantly shift from using your old version to the new version.
+version, invoke the version directly to run tests against it, and, after
+you are satisfied, flip the Alias to point to the new version. Traffic
```
Comment (ombozek) [inline docs/safe_lambda_deployments.rst]: All new traffic

File: docs/safe_lambda_deployments.rst [hunk 5]
```diff
@@ -32,24 +32,24 @@ Instant traffic shifting using Lambda Aliases
...
+will instantly shift from using your old version to using the new version.
```
Comment (ombozek) [inline docs/safe_lambda_deployments.rst]: stick to using your or the but don't switch

File: docs/safe_lambda_deployments.rst (hunk 3 above)
Comment (brettstack) [reply docs/safe_lambda_deployments.rst]: So... @michaeldclifford is literally correct? ;)

**Pull request merged**